### DISCUSSION TITLE
Vite manifest not found

### DISCUSSION THREAD

**Discussion by iDaemonFr** [firefly-iii/firefly-iii#9318] [Firefly III]
### Support guidelines

- [x] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [x] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/explanation/more-information/what-its-not/)
- [x] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been requested before.

### Description

I am trying to install Firefly III by following these steps: https://docs.firefly-iii.org/how-to/firefly-iii/installation/self-managed/

However, I encountered the following error message:

javascript

Uncaught Illuminate\Foundation\ViteManifestNotFoundException: Vite manifest not found at: /var/www/<MYPATH>/public/build/manifest.json in /var/www/<MYPATH>/vendor/laravel/framework/src/Illuminate/Foundation/Vite.php:920 Stack trace: #0

Here’s what I’ve tried so far:

    npm install (working)
    composer install (working)
    npm run prod / dev / build (not working)

_____________________________________________________________________________________________

The error message is as follows:

npm run prod
npm ERR! Missing script: "prod"
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     /<USER>/.npm/_logs/2024-10-06T11_00_30_805Z-debug-0.log

_____________________________________________________________________________________________

The complete error log:

 cat /<USER>/.npm/_logs/2024-10-06T11_00_30_805Z-debug-0.log
0 verbose cli /usr/bin/node /usr/bin/npm
1 info using npm@9.2.0
2 info using node@v18.19.0
3 timing npm:load:whichnode Completed in 0ms
4 timing config:load:defaults Completed in 1ms
5 timing config:load:file:/usr/share/nodejs/npm/npmrc Completed in 1ms
6 timing config:load:builtin Completed in 1ms
7 timing config:load:cli Completed in 1ms
8 timing config:load:env Completed in 0ms
9 timing config:load:file:/var/www/<MYPATH>/.npmrc Completed in 0ms
10 timing config:load:project Completed in 1ms
11 timing config:load:file:/<USER>/.npmrc Completed in 0ms
12 timing config:load:user Completed in 0ms
13 timing config:load:file:/etc/npmrc Completed in 0ms
14 timing config:load:global Completed in 0ms
15 timing config:load:setEnvs Completed in 1ms
16 timing config:load Completed in 7ms
17 timing npm:load:configload Completed in 7ms
18 timing npm:load:mkdirpcache Completed in 0ms
19 timing npm:load:mkdirplogs Completed in 0ms
20 verbose title npm run prod
21 verbose argv "run" "prod"
22 timing npm:load:setTitle Completed in 1ms
23 timing config:load:flatten Completed in 2ms
24 timing npm:load:display Completed in 3ms
25 verbose logfile logs-max:10 dir:/<USER>/.npm/_logs/2024-10-06T11_00_30_805Z-
26 verbose logfile /<USER>/.npm/_logs/2024-10-06T11_00_30_805Z-debug-0.log
27 timing npm:load:logFile Completed in 3ms
28 timing npm:load:timers Completed in 0ms
29 timing npm:load:configScope Completed in 0ms
30 timing npm:load Completed in 14ms
31 silly logfile start cleaning logs, removing 1 files
32 timing config:load:flatten Completed in 0ms
33 silly logfile done cleaning log files
34 timing command:run Completed in 3ms
35 verbose stack Error: Missing script: "prod"
35 verbose stack
35 verbose stack To see a list of scripts, run:
35 verbose stack   npm run
35 verbose stack     at RunScript.run (/usr/share/nodejs/npm/lib/commands/run-script.js:98:13)
35 verbose stack     at async module.exports (/usr/share/nodejs/npm/lib/cli.js:133:5)
36 verbose cwd /var/www/<MYPATH>
37 verbose Linux 6.1.0-25-amd64
38 verbose node v18.19.0
39 verbose npm  v9.2.0
40 error Missing script: "prod"
40 error
40 error To see a list of scripts, run:
40 error   npm run
41 verbose exit 1
42 timing npm Completed in 28ms
43 verbose code 1
44 error A complete log of this run can be found in:
44 error     /<USER>/.npm/_logs/2024-10-06T11_00_30_805Z-debug-0.log

_____________________________________________________________________________________________

Thank you

### Solution

No solution for now

**Comment by JC5** [comment]
Hi, thanks for opening an issue.

Why are you running npm commands? Those are not in the instructions. The necessary files have already been built and you don't need to do that again.
